pkg/tasks/alertmanager.go

```diff
@@ -236,6 +232,16 @@ func (t *AlertmanagerTask) destroy(ctx context.Context) error {
 		return errors.Wrap(err, "deleting Alertmanager RBAC proxy Secret failed")
 	}
 
+	rsw, err := t.factory.AlertmanagerRBACProxyWebSecret()
+	if err != nil {
+		return errors.Wrap(err, "initializing Alertmanager RBAC proxy web Secret failed")
+	}
+
+	err = t.client.DeleteSecret(ctx, rsw)
+	if err != nil {
+		return errors.Wrap(err, "deleting Alertmanager RBAC proxy web Secret failed")
+	}
+
```

Comment:
do we need this since the `create()` function would delete the secret too?

Reply:
This secret is useful for alertmanager. It provides the certificate to kube-rbac-proxy serving REST API calls on port 9095.